client/src/partials/reports/cotisation_payment/cotisation_payment.html

```diff
@@ -20,8 +20,24 @@
                     <span class="input-group-addon">
                       <span class="glyphicon glyphicon-folder-open"></span>
                     </span>  
-                    <select class="form-bhima" ng-model="session.period_id">
-                      <option ng-repeat="getPeriod in getPeriods.data" ng-value="getPeriod.id">{{getPeriod.label}} :: {{getPeriod.dateFrom | date }} - {{getPeriod.dateTo | date }}</option>
+                    <select class="form-bhima" required ng-click="selecCotisations()" ng-model="session.period_id">
```

Comment:
@lomamech, could you use `ng-options` here?  It is much better for performance.  See #701 and #706 .